webhook_server/libs/github_api.py

```
@@ -770,6 +784,51 @@ async def cleanup(self) -> None:
+
+        for check in raw_checks:
+            # Validate name field
+            check_name = check.get("name")
+            if not check_name:
+                self.logger.warning("Custom check missing required 'name' field, skipping")
+                continue
+
+            # Validate command field
+            command = check.get("command")
+            if not command or not command.strip():
+                self.logger.warning(f"Custom check '{check_name}' missing required 'command' field, skipping")
```

Ruff (0.14.10)

Logging statement uses f-string

(G004)